Generators fail to generate templates in windows

### Expected behavior
Generator should generate corresponding templates in commandline 
`choo generate page newpage`  or
`choo generate element newElement`
should trigger generator build

### Actual behavior
generator fails to generate templates but rather it throws an error in windows console. 

```
C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\bin\choo-generate.js:36
  throw e
  ^
AssertionError: Trying to copy from a source that does not exist: C:\Users\rolly_000\WebstormProjects\CWCSProject\client\elements\C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli
\generators\element\template.js
    at EditionInterface.exports._copySingle (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\node_modules\mem-fs-editor\lib\actions\copy.js:45:3)
    at EditionInterface.exports.copy (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\node_modules\mem-fs-editor\lib\actions\copy.js:23:17)
    at EditionInterface.module.exports [as move] (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\node_modules\mem-fs-editor\lib\actions\move.js:4:8)
    at mv (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\lib\utils.js:35:28)
    at generate (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\lib\utils.js:38:3)
    at Object.module.exports [as element] (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\generators\element\index.js:8:3)
    at Object.<anonymous> (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\bin\choo-generate.js:33:23)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\lib\exec.js:10
      throw new Error('Error code: ' + code)
      ^

Error: Error code: 1
    at ChildProcess.exec.on.code (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\lib\exec.js:10:13)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at ChildProcess.cp.emit (C:\Program Files (x86)\Nodist\bin\node_modules\choo-cli\node_modules\cross-spawn\lib\enoent.js:40:29)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)

```


### Steps to reproduce behavior
enter choo generate command in command line e.g
`choo generate page newPage`

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -1,54 +1,1 @@
-const store = require('mem-fs').create()
-const xfs = require('mem-fs-editor').create(store)
-const kebabCase = require('lodash').kebabCase
-const exec = require('../lib/exec')
-const resolvePath = require('path').resolve
-const once = require('ramda').once
-const walkBack = require('walk-back')
-const chalk = require('chalk')
-const delim = ( require('os').platform()==='win32' ) ? '\\' :'/'
-
-const cwd = process.cwd()
-
-const newProjectPath = (projectName) => (dir) => resolvePath(cwd, projectName, dir || '')
-const findConfig = once(() => walkBack(process.cwd(), 'choo.yaml'))
-
-const findRootPath = once(() => {
-  if (process.env.PROJECT_PATH) {
-    return process.env.PROJECT_PATH
-  } else {
-    if (findConfig()) {
-      return findConfig().split(delim).slice(0, -1).join(delim)
-    } else {
-      throw new Error('cannot find choo.yaml from ' + cwd)
-    }
-  }
-})
-
-const destinationPath = (dir) => resolvePath(findRootPath(), dir || '')
-const npmInstall = process.env.NODE_ENV === 'test' ? () => { } : () => process.nextTick(() => {
-  exec('npm', ['install', '-s'], { cwd: destinationPath() })
-})
-
-const generate = (templatePath, category, props) => {
-  const fileName = `${kebabCase(props.name)}.js`
-  const targetPath = resolvePath(findRootPath(), props.path || category)
-  const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))
-  const templateFileName = templatePath.split(delim).slice(-2)[0]
-  xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)
-  mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)
-  xfs.commit(() => {
-    console.log(`  ${chalk.green.bold('generate')}`, chalk.white.bold(`${props.path || category}/${fileName}`))
-  })
-}
-
-module.exports = {
-  xfs,
-  cwd,
-  generate,
-  npmInstall,
-  findConfig,
-  findRootPath,
-  newProjectPath,
-  destinationPath
-}
+const store = require('mem-fs').create()const xfs = require('mem-fs-editor').create(store)const kebabCase = require('lodash').kebabCaseconst exec = require('../lib/exec')const resolvePath = require('path').resolveconst once = require('ramda').onceconst walkBack = require('walk-back')const chalk = require('chalk')const delim = ( require('os').platform()==='win32' ) ? '\\' :'/'const cwd = process.cwd()const newProjectPath = (projectName) => (dir) => resolvePath(cwd, projectName, dir || '')const findConfig = once(() => walkBack(process.cwd(), 'choo.yaml'))const findRootPath = once(() => {  if (process.env.PROJECT_PATH) {    return process.env.PROJECT_PATH  } else {    if (findConfig()) {      return findConfig().split(delim).slice(0, -1).join(delim)    } else {      throw new Error('cannot find choo.yaml from ' + cwd)    }  }})const destinationPath = (dir) => resolvePath(findRootPath(), dir || '')const npmInstall = process.env.NODE_ENV === 'test' ? () => { } : () => process.nextTick(() => {  exec('npm', ['install', '-s'], { cwd: destinationPath() })})const generate = (templatePath, category, props) => {  const fileName = `${kebabCase(props.name)}.js`  const targetPath = resolvePath(findRootPath(), props.path || category)  const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))  const templateFileName = templatePath.split(delim).slice(-2)[0]  xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)  mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)  xfs.commit(() => {    console.log(`${chalk.green.bold('generate')}`, chalk.white.bold(`${props.path || category}/${fileName}`))  })}module.exports = {  xfs,  cwd,  generate,  npmInstall,  findConfig,  findRootPath,  newProjectPath,  destinationPath}

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -1,54 +1,1 @@
-const store = require('mem-fs').create()
-const xfs = require('mem-fs-editor').create(store)
-const kebabCase = require('lodash').kebabCase
-const exec = require('../lib/exec')
-const resolvePath = require('path').resolve
-const once = require('ramda').once
-const walkBack = require('walk-back')
-const chalk = require('chalk')
-const delim=(require('os').platform()==='win32')? '\\' :'/'
-
-const cwd = process.cwd()
-
-const newProjectPath = (projectName) => (dir) => resolvePath(cwd, projectName, dir || '')
-const findConfig = once(() => walkBack(process.cwd(), 'choo.yaml'))
-
-const findRootPath = once(() => {
-  if (process.env.PROJECT_PATH) {
-    return process.env.PROJECT_PATH
-  } else {
-    if (findConfig()) {
-      return findConfig().split(delim).slice(0, -1).join(delim)
-    } else {
-      throw new Error('cannot find choo.yaml from ' + cwd)
-    }
-  }
-})
-
-const destinationPath = (dir) => resolvePath(findRootPath(), dir || '')
-const npmInstall = process.env.NODE_ENV === 'test' ? () => { } : () => process.nextTick(() => {
-  exec('npm', ['install', '-s'], { cwd: destinationPath() })
-})
-
-const generate = (templatePath, category, props) => {
-  const fileName = `${kebabCase(props.name)}.js`
-  const targetPath = resolvePath(findRootPath(), props.path || category)
-  const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))
-  const templateFileName = templatePath.split(delim).slice(-2)[0]
-  xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)
-  mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)
-  xfs.commit(() => {
-    console.log(`  ${chalk.green.bold('generate')}`, chalk.white.bold(`${props.path || category}/${fileName}`))
-  })
-}
-
-module.exports = {
-  xfs,
-  cwd,
-  generate,
-  npmInstall,
-  findConfig,
-  findRootPath,
-  newProjectPath,
-  destinationPath
-}
+const store = require('mem-fs').create()const xfs = require('mem-fs-editor').create(store)const kebabCase = require('lodash').kebabCaseconst exec = require('../lib/exec')const resolvePath = require('path').resolveconst once = require('ramda').onceconst walkBack = require('walk-back')const chalk = require('chalk')const delim = ( require('os').platform()==='win32' )? '\\' :'/'const cwd = process.cwd()const newProjectPath = (projectName) => (dir) => resolvePath(cwd, projectName, dir || '')const findConfig = once(() => walkBack(process.cwd(), 'choo.yaml'))const findRootPath = once(() => {  if (process.env.PROJECT_PATH) {    return process.env.PROJECT_PATH  } else {    if (findConfig()) {      return findConfig().split(delim).slice(0, -1).join(delim)    } else {      throw new Error('cannot find choo.yaml from ' + cwd)    }  }})const destinationPath = (dir) => resolvePath(findRootPath(), dir || '')const npmInstall = process.env.NODE_ENV === 'test' ? () => { } : () => process.nextTick(() => {  exec('npm', ['install', '-s'], { cwd: destinationPath() })})const generate = (templatePath, category, props) => {  const fileName = `${kebabCase(props.name)}.js`  const targetPath = resolvePath(findRootPath(), props.path || category)  const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))  const templateFileName = templatePath.split(delim).slice(-2)[0]  xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)  mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)  xfs.commit(() => {    console.log(`  ${chalk.green.bold('generate')}`, chalk.white.bold(`${props.path || category}/${fileName}`))  })}module.exports = {  xfs,  cwd,  generate,  npmInstall,  findConfig,  findRootPath,  newProjectPath,  destinationPath}

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -6,6 +6,7 @@ const resolvePath = require('path').resolve
 const once = require('ramda').once
 const walkBack = require('walk-back')
 const chalk = require('chalk')
+const delim=(require('os').platform()==='win32')? '\\' :'/'
 
 const cwd = process.cwd()
 
@@ -17,7 +18,7 @@ const findRootPath = once(() => {
     return process.env.PROJECT_PATH
   } else {
     if (findConfig()) {
-      return findConfig().split('/').slice(0, -1).join('/')
+      return findConfig().split(delim).slice(0, -1).join(delim)
     } else {
       throw new Error('cannot find choo.yaml from ' + cwd)
     }
@@ -33,7 +34,7 @@ const generate = (templatePath, category, props) => {
   const fileName = `${kebabCase(props.name)}.js`
   const targetPath = resolvePath(findRootPath(), props.path || category)
   const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))
-  const templateFileName = templatePath.split('/').slice(-2)[0]
+  const templateFileName = templatePath.split(delim).slice(-2)[0]
   xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)
   mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)
   xfs.commit(() => {

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -1,1 +1,54 @@
-const store = require('mem-fs').create()const xfs = require('mem-fs-editor').create(store)const kebabCase = require('lodash').kebabCaseconst exec = require('../lib/exec')const resolvePath = require('path').resolveconst once = require('ramda').onceconst walkBack = require('walk-back')const chalk = require('chalk')const delim = ( require('os').platform()==='win32' )? '\\' :'/'const cwd = process.cwd()const newProjectPath = (projectName) => (dir) => resolvePath(cwd, projectName, dir || '')const findConfig = once(() => walkBack(process.cwd(), 'choo.yaml'))const findRootPath = once(() => {  if (process.env.PROJECT_PATH) {    return process.env.PROJECT_PATH  } else {    if (findConfig()) {      return findConfig().split(delim).slice(0, -1).join(delim)    } else {      throw new Error('cannot find choo.yaml from ' + cwd)    }  }})const destinationPath = (dir) => resolvePath(findRootPath(), dir || '')const npmInstall = process.env.NODE_ENV === 'test' ? () => { } : () => process.nextTick(() => {  exec('npm', ['install', '-s'], { cwd: destinationPath() })})const generate = (templatePath, category, props) => {  const fileName = `${kebabCase(props.name)}.js`  const targetPath = resolvePath(findRootPath(), props.path || category)  const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))  const templateFileName = templatePath.split(delim).slice(-2)[0]  xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)  mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)  xfs.commit(() => {    console.log(`  ${chalk.green.bold('generate')}`, chalk.white.bold(`${props.path || category}/${fileName}`))  })}module.exports = {  xfs,  cwd,  generate,  npmInstall,  findConfig,  findRootPath,  newProjectPath,  destinationPath}
+const store = require('mem-fs').create()
+const xfs = require('mem-fs-editor').create(store)
+const kebabCase = require('lodash').kebabCase
+const exec = require('../lib/exec')
+const resolvePath = require('path').resolve
+const once = require('ramda').once
+const walkBack = require('walk-back')
+const chalk = require('chalk')
+const delim = ( require('os').platform()==='win32' )? '\\' :'/'
+
+const cwd = process.cwd()
+
+const newProjectPath = (projectName) => (dir) => resolvePath(cwd, projectName, dir || '')
+const findConfig = once(() => walkBack(process.cwd(), 'choo.yaml'))
+
+const findRootPath = once(() => {
+  if (process.env.PROJECT_PATH) {
+    return process.env.PROJECT_PATH
+  } else {
+    if (findConfig()) {
+      return findConfig().split(delim).slice(0, -1).join(delim)
+    } else {
+      throw new Error('cannot find choo.yaml from ' + cwd)
+    }
+  }
+})
+
+const destinationPath = (dir) => resolvePath(findRootPath(), dir || '')
+const npmInstall = process.env.NODE_ENV === 'test' ? () => { } : () => process.nextTick(() => {
+  exec('npm', ['install', '-s'], { cwd: destinationPath() })
+})
+
+const generate = (templatePath, category, props) => {
+  const fileName = `${kebabCase(props.name)}.js`
+  const targetPath = resolvePath(findRootPath(), props.path || category)
+  const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))
+  const templateFileName = templatePath.split(delim).slice(-2)[0]
+  xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)
+  mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)
+  xfs.commit(() => {
+    console.log(`  ${chalk.green.bold('generate')}`, chalk.white.bold(`${props.path || category}/${fileName}`))
+  })
+}
+
+module.exports = {
+  xfs,
+  cwd,
+  generate,
+  npmInstall,
+  findConfig,
+  findRootPath,
+  newProjectPath,
+  destinationPath
+}

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -6,7 +6,7 @@ const resolvePath = require('path').resolve
 const once = require('ramda').once
 const walkBack = require('walk-back')
 const chalk = require('chalk')
-const delim = ( require('os').platform()==='win32' )? '\\' :'/'
+const delim = ( require('os').platform()==='win32' ) ? '\\' :'/'
 
 const cwd = process.cwd()
 

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -6,7 +6,7 @@ const resolvePath = require('path').resolve
 const once = require('ramda').once
 const walkBack = require('walk-back')
 const chalk = require('chalk')
-const delim = (require('os').platform() === 'win32') ? '\\' :'/'
+const delim = (require('os').platform() === 'win32') ? '\\' : '/'
 
 const cwd = process.cwd()
 

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -1,1 +1,51 @@
-const store = require('mem-fs').create()const xfs = require('mem-fs-editor').create(store)const kebabCase = require('lodash').kebabCaseconst exec = require('../lib/exec')const resolvePath = require('path').resolveconst once = require('ramda').onceconst walkBack = require('walk-back')const chalk = require('chalk')const delim = ( require('os').platform()==='win32' ) ? '\\' :'/'const cwd = process.cwd()const newProjectPath = (projectName) => (dir) => resolvePath(cwd, projectName, dir || '')const findConfig = once(() => walkBack(process.cwd(), 'choo.yaml'))const findRootPath = once(() => {  if (process.env.PROJECT_PATH) {    return process.env.PROJECT_PATH  } else {    if (findConfig()) {      return findConfig().split(delim).slice(0, -1).join(delim)    } else {      throw new Error('cannot find choo.yaml from ' + cwd)    }  }})const destinationPath = (dir) => resolvePath(findRootPath(), dir || '')const npmInstall = process.env.NODE_ENV === 'test' ? () => { } : () => process.nextTick(() => {  exec('npm', ['install', '-s'], { cwd: destinationPath() })})const generate = (templatePath, category, props) => {  const fileName = `${kebabCase(props.name)}.js`  const targetPath = resolvePath(findRootPath(), props.path || category)  const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))  const templateFileName = templatePath.split(delim).slice(-2)[0]  xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)  mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)  xfs.commit(() => {    console.log(`${chalk.green.bold('generate')}`, chalk.white.bold(`${props.path || category}/${fileName}`))  })}module.exports = {  xfs,  cwd,  generate,  npmInstall,  findConfig,  findRootPath,  newProjectPath,  destinationPath}
+const store = require('mem-fs').create()
+const xfs = require('mem-fs-editor').create(store)
+const kebabCase = require('lodash').kebabCase
+const exec = require('../lib/exec')
+const resolvePath = require('path').resolve
+const once = require('ramda').once
+const walkBack = require('walk-back')
+const chalk = require('chalk')
+const delim = (require('os').platform() === 'win32') ? '\\' :'/'
+
+const cwd = process.cwd()
+
+const newProjectPath = (projectName) => (dir) => resolvePath(cwd, projectName, dir || '')
+const findConfig = once(() => walkBack(process.cwd(), 'choo.yaml'))
+const findRootPath = once(() => {
+  if (process.env.PROJECT_PATH) {
+    return process.env.PROJECT_PATH
+  } else {
+    if (findConfig()) {
+      return findConfig().split(delim).slice(0, -1).join(delim)
+    } else {
+      throw new Error('cannot find choo.yaml from ' + cwd)
+    }
+  }
+})
+const destinationPath = (dir) => resolvePath(findRootPath(), dir || '')
+const npmInstall = process.env.NODE_ENV === 'test' ? () => { } : () => process.nextTick(() => {
+  exec('npm', ['install', '-s'], { cwd: destinationPath() })
+})
+
+const generate = (templatePath, category, props) => {
+  const fileName = `${kebabCase(props.name)}.js`
+  const targetPath = resolvePath(findRootPath(), props.path || category)
+  const mv = (a, b) => xfs.move(destinationPath(a), destinationPath(b))
+  const templateFileName = templatePath.split(delim).slice(-2)[0]
+  xfs.copyTpl(`${templatePath}/**`, destinationPath(targetPath), props)
+  mv(`${props.path || category}/${templateFileName}.js`, `${props.path || category}/${fileName}`)
+  xfs.commit(() => {
+    console.log(`${chalk.green.bold('generate')}`, chalk.white.bold(`${props.path || category}/${fileName}`))
+  })
+}
+module.exports = {
+  xfs,
+  cwd,
+  generate,
+  npmInstall,
+  findConfig,
+  findRootPath,
+  newProjectPath,
+  destinationPath
+}

